scripts/cli/cli_helper.py

```diff
@@ -1,38 +1,53 @@
 import argparse
 import json
+import os
 from datetime import datetime
 from os import environ
-from typing import List, Optional
+from typing import List, NamedTuple, Optional
 
 from dateutil import parser, tz
 from linz_logger import get_log
 
+from scripts.files.files_helper import get_file_name_from_path
 
-def format_source(source: List[str]) -> List[str]:
-    """Due to Argo constraints if using the basemaps cli list command
-    the source has a string that contains a list that needs to be split.
-    example: ["[\"s3://test/image_one.tiff\", \"s3://test/image_two.tiff\"]"]
+
+class TileFiles(NamedTuple):
+    output: str
+    input: List[str]
+
+
+def format_source(source: List[str]) -> List[TileFiles]:
```

Comment:
This will need some improvement to handle better local run (dev/debug) [TDE-821]